feat: allow http calls without origin header (#1941)
tauri-apps/plugins-workspace: .changes/http-allow-skip-origin.md

---
"http": "patch"
"http-js": "patch"
---

> Allow skipping sending Origin header in HTTP requests by sending empty string as the value for Origin header.

Reviewer: Just a slight wording change and this should be ready to go
```suggestion
Allow skipping sending `Origin` header in HTTP requests by setting `Origin` header to an empty string when calling `fetch`.
```
